lib/puppet/type/selboolean.rb

```diff
@@ -9,9 +9,23 @@ module Puppet
     end
 
     newproperty(:value) do
-      desc "Whether the SELinux boolean should be enabled or disabled."
-      newvalue(:on)
-      newvalue(:off)
+      VALUE_DOC="Valid values are: \"on\"/\"true\"/\"off\"/\"false\""
```

Comment:
you can wrap this string in single quotes to avoid escaping the double quotes inside

also, can you put a whitespace before and after the `=` sign